deepinv/optim/fixed_point.py

```diff
@@ -242,7 +242,6 @@ def forward(self, *args, compute_metrics=False, x_gt=None, **kwargs):
             range(self.max_iter),
             disable=(not self.verbose or not self.show_progress_bar),
         ):
-
```

Comment:
Why are there lots of changed files with newlines removed?

Reply:
It's was due to black formatter. Also, as I made some improvements on SpaceVaryingBlurGenerator, which lead to some neccesary changes in SpaceVaryingBlur physics. The changes is mainly  about the documentation.

Reply:
And I forgot to turn off my autopep8 formatter so sometimes it changes `lambda x: ....` into `def func(x): ...`

Reply:
Please can we remove the extra newlines changes as they aren't relevant to this PR?

Reply:
The extra changes came from when I merged this branch with main. I'll remove it as you suggested